UpSkill_App/UpSkill.Infrastructure/Data/Models/Company.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+
+    using static DataConstants.Company;
+
+    public class Company
+    {
+        [Key]
+        [Required]
+        public string Id { get; init; } = Guid.NewGuid().ToString();
+
+        [Required]
+        public string Name { get; set; }
+        
+        [Required]
+        public string OwnerId { get; set; }
+        public Owner Owner { get; set; }
+        
+        [Required]
```

Comment:
Not sure I recognize the abbreviation? What does it stand for?

Reply:
Which abbreviation?